plugins/tiddlywiki/multiwikiclient/multiwikiclientadaptor.js

```diff
@@ -0,0 +1,341 @@
+/*\
+title: $:/plugins/tiddlywiki/multiwikiclient/multiwikiclientadaptor.js
+type: application/javascript
+module-type: syncadaptor
+
+A sync adaptor module for synchronising with MultiWikiServer-compatible servers
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+var CONFIG_HOST_TIDDLER = "$:/config/multiwikiclient/host",
```

Comment:
In other code below you used `const` -- IMO it could be used here too.

Reply:
Just saw this video about const -- [Let Me Be: "const ... Is Stupid and You're Using It Wrong"](https://www.epicweb.dev/talks/let-me-be) 

I think it's worth viewing.

Reply:
> I think it's worth viewing.

I'm afraid I do not generally watch videos, could you summarise the key points?